python/lsst/meas/algorithms/objectSizeStarSelector.py

```diff
@@ -410,13 +434,18 @@ def selectSources(self, sourceCat, matches=None, exposure=None):
         width = numpy.sqrt(0.5*(xx + yy))
         with numpy.errstate(invalid="ignore"):  # suppress NAN warnings
             bad = reduce(lambda x, y: numpy.logical_or(x, sourceCat.get(y)), self.config.badFlags, False)
-            bad = numpy.logical_or(bad, flux < self.config.fluxMin)
             bad = numpy.logical_or(bad, numpy.logical_not(numpy.isfinite(width)))
             bad = numpy.logical_or(bad, numpy.logical_not(numpy.isfinite(flux)))
+            if self.config.doFluxLimit:
```

Comment:
Perhaps it would be better to set the Min/Max values to a default of `None` and test if a value has been set rather than using the extra boolean variables. For instance

    if self.config.fluxMin is not None:
        bad = numpy.logical_or(bad, flux < self.config.fluxMin)